lib/assets/javascripts/turbograft/page.coffee

```diff
@@ -15,18 +15,15 @@ Page.refresh = (options = {}, callback) ->
     location.pathname + paramString
   else
     location.href
+  delete options.url
 
+  options.partialReplace = true
+  options.callback = callback if callback
   if options.response
-    options.partialReplace = true
-    options.onLoadFunction = callback
-
     xhr = options.response
     delete options.response
```

Comment:
I just noticed this is here, does this remove references properly? Pretty sure `delete` will leave any references to the object deleted. Is setting `xhr` to `options.response` and then deleting it leaking?